src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,99 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: string[],
+  perPage: number,
+  currentPage: number,
+  onPageChange: (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pageItems = total.slice(
+    (currentPage - 1) * perPage, currentPage * perPage,
+  );
+
+  const pagesTotal = getNumbers(1, Math.ceil(total.length / perPage));
+
+  const disablePrevPage = currentPage === 1;
+  const disableNextPage = currentPage === pagesTotal[pagesTotal.length - 1];
```

Comment:
could you use a common isButtonDisabled for both buttons?
```suggestion
   const isButtonDisabled  = currentPage === 1 || currentPage === pagesTotal[pagesTotal.length - 1];
```